Sources/CircularProgress/CircularProgress.swift

```diff
@@ -415,3 +371,60 @@ public final class CircularProgress: NSView {
 		progressLabel.isHidden = !cancelButton.isHidden
 	}
 }
+
+extension CircularProgress:MouseTrackable{
+	public override func hitTest(_ point: NSPoint) -> NSView? {
+		guard let mousePos = self.window?.mouseLocationOutsideOfEventStream else {return nil}
+		let localMousePos = self.convert(mousePos,from:nil)
+		if let path = path {
+			let contains:Bool = path.contains(localMousePos)
+			return contains ? self : nil
+		}
+		return super.hitTest(point)
+	}
+	var path: CGPath? { return CGPath.init(ellipseIn: bounds.insetBy(dx: 20, dy: 20), transform: nil) }
+	override public func updateTrackingAreas() {
+		guard isCancellable else {
+			return
+		}
+		createTrackingArea([.activeAlways, .mouseMoved, .mouseEnteredAndExited])
```

Comment:
Why `.activeAlways`?

Reply:
I guess it can be replaced with NSTrackingActiveInActiveApp. Or maybe removed all together? I'll have a look.

Reply:
✅ changed to .activeInApp instead.